torso-bundle.js

```diff
@@ -681,6 +683,7 @@
             options = options || {};
             //find ids that we don't have in cache
             options.idsToFetch = _.difference(this.getTrackedIds(), _.pluck(parentInstance.models, 'id'));
+            options.idsToFetch = _.difference(options.idsToFetch, _.uniq(_.flatten(_.values(parentInstance.idsFetching))));
```

Comment:
Same question about whether you're trying to overwrite or append to options.idsToFetch.